packages/vite/src/node/plugins/css.ts

```diff
@@ -1428,7 +1433,7 @@ async function compileCSS(
     )
   }
 
-  if (!postcssPlugins.length) {
+  if (lang !== 'sss' && !postcssPlugins.length) {
```

Comment:
This `lang !== 'sss'` was not needed because `!postcssPlugins.length` was always `false`.